### ISSUE TITLE
Create new digit images.

### ISSUE THREAD

**Issue by pfarmer** [pfarmer/jquery-countdown#8]

**Comment by ghost** [comment]
I created a smaller version of the original transparent version (27x39) because the ones I found were too choppy/ didnt work well.

http://s16.postimage.org/bwsbds1tf/digits2.png.

Hope it helps!

**Comment by machonemedia** [comment]
These calculations from the initial project work great for resizing:

Example how to change size:
The PNG consists of 60 frames, so if you want your Height to be 18px you multiply 18 by 60 = 1080px. Then you divide 4619 (the original Height of the PNG) into 1080 = 4,276851851851852. Then you divide 53 (the original Width of the PNG) into 4,276851851851852 = 12,39229270404849 and you round it up or down (12 or 13).

So if you want to achieve the following size:
digitWidth: 13,
digitHeight: 18
You will have to resize your image to 13x1080px.

**Comment by siburny** [comment]
@0sm3l Thanks so much, worked beautiful! You should create a pull request to add that file.

**Comment by FDiskas** [comment]
Then the countdown is in reverse order. Digits goes from 9 to 0, then the animation is in wrong direction. Needs an image with transition to reverse order.

**Comment by siburny** [comment]
Well, actually if you reverse animation it works fine: scroll image from bottom up. Take a look at my working timer: http://www.maximandinna.com/

**Comment by FDiskas** [comment]
Yes but animation shows that the numbers goes from bottom to top. This is
wrong. That type of clock must drop numbers from the top.

**Comment by siburny** [comment]
Think about it: when you implement normal clock, then the numbers go from 0 to 9 and they do drop from the top. But if you do reverse, then number go from 9 to 0 and instead of dropping they should go "up".

PS if I had physical clock like that, I would demonstrate it on the real thing.

**Comment by FDiskas** [comment]
Yes the physics drops from top everythink to earth center in our case
"down" :) sorry for my bad english

**Comment by siburny** [comment]
Yes, it applies to a "normal" flow of the clock :) but if you reverse it (countdown in our case), then it should flip "up"

Anyway, you can create your own image and make it flip sideways :-D
